### PR TITLE
fix(sass): Fix conversion rule for fade() to rgba()

### PR DESCRIPTION
**Pull request by TheRealJon**
## Description
Add new custom Sass conversion rule that converts less fade() function with equivalent sass rgba() function, and properly handles decimal percentages.

Fixes Issue #879, replaces PR #880

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -416,6 +416,12 @@ module.exports = function (grunt) {
               replacement: '$1@include $2()',
               order: 3
             },
+            // Original fade -> rgba conversion did not account for decimal percentages
+            {
+              pattern: /fade\((.*),\s?([\d\.]+)\%\)/gmi,
+              replacement: 'rgba($1, ($2/100))',
```
Comment (jeff-phillips-18) [inline Gruntfile.js:422]: Will we still handle fade that does not use a percentage?

**Review by AllenBW — APPROVED** [review]
Thanks for the quick fix!!

File: Gruntfile.js (hunk 1 above)
Comment (TheRealJon) [reply Gruntfile.js:422]: @jeff-phillips-18 The less fade function signature requires a percentage, so this shouldn't be an issue.

Comment (jeff-phillips-18) [reply Gruntfile.js:422]: Ok great.

**Review by jeff-phillips-18 — APPROVED** [review]

**Pull request merged**